tests/qos/conftest.py

```diff
@@ -0,0 +1,14 @@
+from .args.qos_sai_args import add_qos_sai_args
+
+# WR-ARP pytest arguments
+def pytest_addoption(parser):
```

Comment:
From which folder do you expect to run this test cases? I'm asking because looks like if to use "pytest_addoption" hook not inside main conftest file and in the same time run tests from "sonic-mgmt/tests" directory, pytest will raise error that specified options are not known like you define above.
For example if to pass to the console "--qos_dst_ports" option and run tests from "sonic-mgmt/tests" folder.

@tahmed-dev @wangxin What do you think?

Reply:
I am expecting the test case to run from tests dir. I did not see an issue with passing argument when addoption is called in sub-conftest files. https://docs.pytest.org/en/2.7.3/plugins.html
Did you have issues running the test? I just ran it with no problem using
```
pytest qos/test_qos_sai.py --testbed=vms12-t0-s6000-1 --inventory=../ansible/str --testbed_file=../ansible/testbed.csv --host-pattern=str-s6000-acs-14 --module-path=../ansible/library --disable_loganalyzer --skip_sanity --qos_dst_ports='[0, 2, 3]' --qos_src_ports='[1]'
================================================================================================================================================== test session starts ===================================================================================================================================================
platform linux2 -- Python 2.7.12, pytest-4.6.9, py-1.8.1, pluggy-0.13.1
ansible: 2.8.7
rootdir: /var/host-acs-mgmt-repo/tests, inifile: pytest.ini
plugins: ansible-2.2.2
collected 1 item                                                                                                                                                                                                                                                                                                         

qos/test_qos_sai.py 
```

Reply:
@yvolynets-mlnx There was an issue of reading list from the command line that I fixed in a recent commit. Please try it out and let me know if you hit any issues.